docs/source/development/add-ons/extend-webpack-add-on.md

```diff
@@ -0,0 +1,50 @@
+---
+myst:
+  html_meta:
+    "description": "Extend webpack setup from an add-on with razzle.extend.js"
+    "property=og:description": "Extend webpack setup from an add-on with razzle.extend.js"
+    "property=og:title": "Extend webpack setup from an add-on"
+    "keywords": "Volto, Plone, webpack, add-on, razzle.extend.js, Razzle"
+---
+
+# Extend webpack setup from an add-on
+
+Just like you can extend Razzle's configuration from the project, you can do the same with an add-on.
```

Comment:
This sentence seems outdated...it assumes the reader is familiar with projects, but that's a concept we're trying to get rid of.

Reply:
Nice catch. There may be this and other artifacts that should be removed or updated, perhaps with a `{deprecated}` admonition.

Reply:
In fact, reading this page, I don't think anything in it still applies. @sneridagh can you take a look at this one?

Reply:
Addressed in 4b9f837bd1276ddae9414132f24d605f4970bf60. @davisagli @sneridagh please have a look.